src/ximea_driver.cpp

```diff
@@ -0,0 +1,62 @@
+#include "ximea_driver/ximea_driver.hpp"
+
+#include "sensor_msgs/image_encodings.hpp"
+
+namespace trailbot {
+
+constexpr int kTimeoutMs = 100;
+
+constexpr int kExposureMs = 50;
+constexpr int kExposureUs = kExposureMs * 1000;
```

Comment:
Some clarity on what these values do may be helpful

Reply:
Will add a comment